docs/source/examples/merge_dataset.rst

```diff
@@ -0,0 +1,67 @@
+###############
+ Merge Dataset
+###############
+
+This topic describes the merge dataset operation:
+
+Take the `Oxford-IIIT Pet <https://gas.graviti.cn/dataset/data-decorators/OxfordIIITPet>`_
+and `Dogs vs Cats <https://gas.graviti.cn/dataset/data-decorators/DogsVsCats>`_
+as examples. Their structure looks like::
+
+    Oxford-IIIT Pet/
+        test/
+            Abyssinian_002.jpg
+            ...
+        trainval/
+            Abyssinian_001.jpg
+            ...
+    Dogs vs Cats/
+        test/
+            1.jpg
+            10.jpg
+            ...
+        train/
+            cat.0.jpg
+            cat.1.jpg
+            ...
+
+There are lots of pictures of cats and dogs in these two datasets,
+Users can merge them into one dataset and use the merged dataset for research.
+
+.. note::
+
+   Before merge datasets operation, fork them first.
+
+Create a dataset which is named ``mergeDataset``
+
+.. literalinclude:: ../../../docs/code/merge_dataset.py
+   :language: python
+   :start-after: """Create Target Dataset"""
+   :end-before: """"""
+
+Copy all segments in ``OxfordIIITPetDog`` to ``mergeDataset``
+
+.. literalinclude:: ../../../docs/code/merge_dataset.py
+   :language: python
+   :start-after: """Copy Segment from Pet"""
+   :end-before: """"""
+
+
+Unify categories of ``train`` segment.
+
+.. literalinclude:: ../../../docs/code/merge_dataset.py
+   :language: python
+   :start-after: """Unify category"""
+   :end-before: """"""
+
+.. note::
+
+    The category in ``OxfordIIITPet`` is like ``cat.Abyssinian``, but in ``Dogs vs Cats`` is like
+    ``cat``, thus we need to rename ``cat.Abyssinian`` to ``cat`` to unify them.
```

Comment:
The category in ``OxfordIIITPet`` is of two-level formats, like ``cat.Abyssinian``, but in ``Dogs vs Cats`` it only has one level, like ``cat``, thus we need to unify the categories, for example, rename ``cat.Abyssinian`` to ``cat``.